draft-ietf-rats-corim.md

```diff
@@ -2607,6 +2618,28 @@ If no entry is found, the comparison MUST return false.
 Instead, if an entry is found, the digest comparison proceeds as defined in {{sec-cmp-digests}} after equivalence has been found according to {{sec-comid-integrity-registers}}.
 Note that it is not required for all the entries in the candidate entry to be used during matching: the condition ECT could consist of a subset of the device's register space. In TPM parlance, a TPM "quote" may report all PCRs in Evidence, while a condition ECT could describe a subset of PCRs.
 
+##### Comparison for raw-integral entries
+
+The ACS entry value stored under `measurement-values-map` codepoint 15 is a raw integral value, which must have type `raw-integral-type-choice`.
+
+Consider an `integral` ACS entry value named ENTRY in a `measurement-values-map` codepoint (e.g., 15) that allows comparing `integral` against a either another `integral` or an `integral-range` named CONDITION.
```

Comment:
```suggestion
Consider an `integral` ACS entry value named ENTRY in a `measurement-values-map` codepoint (i.e., 15) that allows comparing `integral` against either another `integral` or an `integral-range` named CONDITION.
```

Reply:
This is not the intention for int-range. Any entry that chooses to use tagged-int-range (i.e., "allows comparing") is to use these comparison rules.